<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="VibinWorkStation" failures="1" tests="6" name="Personal Loan" time="0.067" errors="0" timestamp="2024-04-20T17:06:38 IST">
  <testcase classname="test.day1" name="demo" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.demo(day1.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="test.day2" name="fourthDemo" time="0.002"/>
  <testcase classname="test.day2" name="thirdDemo" time="0.001"/>
  <testcase classname="test.day4" name="LoginAPIHomeLoan" time="0.001"/>
  <testcase classname="test.day4" name="MobileLoginHomeLoan" time="0.002"/>
  <testcase classname="test.day4" name="WebLoginHomeLoan" time="0.001"/>
  <testcase classname="test.day1" name="secondDemo" time="0.0">
    <ignored/>
  </testcase> <!-- secondDemo -->
</testsuite> <!-- Personal Loan -->
